lib/Byng/Pimcore/Sitemap/SitemapPlugin.php

```diff
@@ -52,22 +54,51 @@ public function init()
      */
     public static function install()
     {
-        return true;
+        if (!SitemapPlugin::isInstalled()) {
+            $data = [
+                "key" => "sitemap_exclude",
+                "name" => "Sitemap: Exclude page",
+                "description" => "Add this property to exclude a page from the sitemap",
+                "ctype" => "document",
+                "type" => "bool",
+                "inheritable" => true,
+                "data" => true
+            ];
+            $property = Predefined::create();
+            $property->setValues($data);
+
+            $property->save();
+
+            return "Sitemap plugin successfully installed";
+        }
+
+        return "There was a problem during the installation";
     }
 
     /**
      * {@inheritdoc}
      */
     public static function uninstall()
     {
-        return true;
+        if (SitemapPlugin::isInstalled()) {
+            $property = Predefined::getByKey("sitemap_exclude");
+            $property->delete();
+
+            return "Sitemap plugin is successfully uninstalled";
+        }
+
+        return "There was an error";
     }
 
     /**
      * {@inheritdoc}
      */
     public static function isInstalled()
     {
-        return true;
+        $property = Predefined::getByKey("sitemap_exclude");
+        if ($property->getId()) {
```

Comment:
I'm just wondering, if it changes in the future to be just null, it's not gonna hurt to add it, and might save a few headaches later :)